html/webpack.config.js

```diff
@@ -27,7 +38,29 @@ const baseConfig = {
             },
             {
                 test: /\.s?[ac]ss$/,
-                use: [devMode ? 'style-loader' : MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader'],
+                use: [
+                  devMode ? 'style-loader' : MiniCssExtractPlugin.loader,
+                  'css-loader',
+                  {
+                      loader: 'sass-loader',
+                      options: {
+                          additionalData: inlineFont()
+                     },
+                  },
+                ],
+            },
+            {
+                test: /xterm-addon-image-worker/,
```

Comment:
Remove this block, it has nothing to do with your PR.

Reply:
ah I was wondering about that but couldn't remember. I will remove it right now.